app/docker-compose.yaml

```diff
@@ -1,30 +1,38 @@
-version: '3.9'
+version: "3.9"
 services:
     db:
-      image: postgres:14
-      ports:
-        - "5432:5432"
-      expose:
-        - "5432"
-      env_file: .env
-      volumes:
-        - db:/var/lib/postgresql/data
-        - ./seed:/docker-entrypoint-initdb.d
-      healthcheck:
-        test: [ "CMD-SHELL", "pg_isready -U postgres && psql --username=postgres --list" ]
-        interval: 1s
-        timeout: 20s
-        retries: 20
+        image: postgres:14
+        ports:
+            - "5432:5432"
+        expose:
+            - "5432"
+        env_file: .env
+        volumes:
+            - wordle:/var/lib/postgresql/data
+            - ./seed:/docker-entrypoint-initdb.d
+        healthcheck:
+            test:
+                [
+                    "CMD-SHELL",
+                    "pg_isready -U postgres && psql --username=postgres --list",
+                ]
+            interval: 1s
+            timeout: 20s
+            retries: 20
     web:
         build: .
-        image: app
+        image: node:lts
+        user: "node"
+        working_dir: /home/node/app
         env_file: .env
         environment:
-          - NODE_ENV=production
-#        volumes:
-#            - .:/app
+            - NODE_ENV=development
         depends_on:
-            db:
-              condition: service_healthy
+            - db
+        ports:
+            - "3000:3000"
+        volumes:
+            - ./:/home/node/app
+        command: "npm start"
 volumes:
-  db:
+    wordle:
```

Comment:
Here you are just overriding some of the stuff in the `Dockercompose`. We need the production container since we will need to deploy this project. 

I will create a development `Dockerfile` today - for now, run it locally without docker (only use docker for the `db`; check the `README.md`).

Also, changing the volume name to `wordle` is confusing since it's only being used for the `db` service.